src/content/app/entity-viewer/state/gene-view/transcripts/geneViewTranscriptsSlice.ts

```diff
@@ -67,6 +68,7 @@ export type Filters = Record<string, Filter>;
 
 const defaultStatePerGene: TranscriptsStatePerGene = {
   expandedIds: [],
+  accordionUserInteraction: false,
```

Comment:
I would prefer a better name (here and for the corresponding selector). For one, `accordionUserInteraction` doesn't suggest that this is going to be a boolean; but just as importantly, this flag isn't even tracking user interactions (you run `toggleTranscriptInfo` automatically in `useEffect` to expand the first transcript by default). Maybe `expandedTranscriptsModified`?